Makefile

```diff
@@ -103,11 +103,13 @@ prepare: | prepare\:images up
 install: | prepare
 	$(call message,$(PROJECT_NAME): Installing Drupal)
 	sleep 5
-	$(call docker-www-data, php drush -r /var/www/html/web site-install falcon \
+	$(call docker-www-data, php drush -r web site-install falcon \
 		--db-url=mysql://$(DB_USER):$(DB_PASSWORD)@$(DB_HOST)/$(DB_NAME) --site-name=$(PROJECT_NAME) --account-pass=admin \
 		install_configure_form.enable_update_status_module=NULL --yes)
 	@if [ $(ENV) = "development" ]; then \
 		$(MAKE) -s drush en $(DEVELOPMENT_MODULES); \
+		$(call docker-wodby, php cp web/sites/example.settings.local.php web/sites/default/settings.local.php); \
+		$(call docker-wodby, php sed -i \"/settings.local.php';/s/# //g\" web/sites/default/settings.php); \
```

Comment:
There were 2 ways to implement it:
- just set a config variable system.logging -> error.level
- utilize the development settings supplied in example.settings.local.php

Went with the second option - we can later supply our own version of local settings if needed.